README.md

```diff
@@ -1 +1,132 @@
-# codacy-metrics-scala-seed
\ No newline at end of file
+
+# Codacy Metrics Scala Seed
+
+We use external tools at Codacy, this is the library we use across the multiple external tools integrations.
+For more details and examples of tools that use this project, you can check
+`TODO`.
+
+### Usage
+
+Add to your SBT dependencies:
+
+```
+"com.codacy" %% "codacy-metrics-scala-seed" % "1.0.0-SNAPSHOT"
```

Comment:
Use `<VERSION>`  and then put a maven badge here.

Reply:
✅ Will add the maven badge when it is published.